extensions/omni_vfs/docs/reference.md

```diff
@@ -152,6 +152,64 @@ This backend can be created by invoking `omni_vfs.local_fs(mount)`, where `mount
     under which conditions mounting of new directories is possible, and which mounts can be accessed under given
     conditions.
 
+### `omni_vfs.table_fs` (table backed file system)
+
+This backend can be created by invoking `omni_vfs.table_fs('fs')`, where `fs` is the name
+of the filesystem. The function returns `omni_vfs.table_fs` type which only contains an
+identifier of the instance that references `omni_vfs.table_fs_filesystems` table.
+
+There are two tables for this backend:
+
+* `omni_vfs.table_fs_files` for storing filesystem id, filename, kind and omni_vfs(`update` queries not allowed).
+
+* `omni_vfs.table_fs_file_data` for storing the data and timestamps of `file` kind files.
+
+#### File operation queries
+
+* The following query creates an entry for sample.txt file in `omni_vfs.table_fs_files` in `fs` table_fs filesystem. There is no need to create parent directories, they will be
+created automatically if it doesn't exist.
+```postgresql
+insert into omni_vfs.table_fs_files (filesystem_id, filename, kind)
+values 
+((omni_vfs.table_fs('fs')).id, '/dir/sample.txt', 'file');
+```
+
+* To associate data with the file created above run the following query. It creates an entry in `omni_vfs.table_fs_file_data`. A utility function named `omni_vfs.table_fs_file_id` is provided to obtain file_id given a table_fs filesystem and filename. Only a single data entry can be associated with a given file.
```

Comment:
This illuminates the fact that we need write side of VFS API, too :)

Reply:
Yes and probably removal of files as well